aten/src/ATen/core/dynamic_type.cpp

```diff
@@ -271,6 +277,16 @@ TypePtr DynamicType::fallback() const {
       return VarType::create(*name_);
     case Tag::AnyClass:
       return AnyClassType::get();
+    case Tag::QScheme:
```

Comment:
Ah, looks like it's rebased changes. Don't know why it shows up here. Is there any way to filter them out for easier review?